<ui version="4.0" >
 <author>Volker Krause &lt;vkrause@kde.org></author>
 <class>SubscriptionDialog</class>
 <widget class="QWidget" name="SubscriptionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>525</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="5" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Manage which folders you want to see in the folder tree</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Search:</string>
       </property>
       <property name="buddy" >
        <cstring>klineedit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="klineedit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="clickMessage" >
        <string>TODO</string>
       </property>
       <property name="showClearButton" stdset="0" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;b>Available folders&lt;/b></string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&lt;b>Current changes&lt;/b></string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="3" column="0" colspan="2" >
    <widget class="QTreeView" name="collectionView" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="KPushButton" name="subscribeButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Subscribe to selected folder</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2" >
    <widget class="QTreeView" name="subscribeView" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="KPushButton" name="unsubscribeButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Unsubscribe from selected folder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2" >
    <widget class="QTreeView" name="unsubscribeView" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes/>
 <resources/>
 <connections/>
</ui>
